docs/content/doc/installation/run-as-service-in-ubuntu.en-us.md

```diff
@@ -31,8 +28,7 @@ PORT or remove the -p flag if default port is used.
 
 Enable and start Gitea at boot:
 ```
-sudo systemctl enable gitea
-sudo systemctl start gitea
+sudo systemctl enable gitea --now
```

Comment:
@bagasme  eafen if its a shortcut; I would leave it so people understand what they do